server/database/fixtures/ArtSeeder.js

```diff
@@ -1,11 +1,38 @@
+/* eslint-disable object-shorthand */
 const AbstractSeeder = require("./AbstractSeeder");
 
+function getRandomCoordinates(centerLat, centerLng, radius) {
```

Comment:
good thing! I'll use this for the admin validation feature

Reply:
Nice 👍